pkg/plugins/golang/v3/scaffolds/internal/templates/api/webhook_suitetest.go

```diff
@@ -163,7 +163,8 @@ var _ = BeforeSuite(func() {
 
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join({{ .BaseDirectoryRelativePath }}, "config", "crd", "bases")},
+		CRDDirectoryPaths:     []string{filepath.Join({{ .BaseDirectoryRelativePath }}, "config", "crd", "bases")},
+		ErrorIfCRDPathMissing: false,
```

Comment:
Could we create a variable to pass here and add on top of this var in the sea above the following comment?

// todo: currently is not possible to know if an API was or not scaffolded. We can fix it when  https://github.com/kubernetes-sigs/kubebuilder/issues/1826 be addressed

Reply:
done, i'm sure WireResource is a bad name in webhook, but it's closest to standard i can see.